src/components/UserItem.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+	<li @click="onClick" :class="[activeId == user.id ? `active` : '']">
+		{{ fullName }}
+	</li>
+</template>
+
+<script>
+export default {
+	props: {
+		user: Object,
+		activeId: null || Number,
```

Comment:
try to avoid such props definition [here](https://vuejs.org/v2/guide/components-props.html) you can find a good guide
![image](https://user-images.githubusercontent.com/34798695/128321458-dcc64ea5-8b59-4a58-bbc9-f0efd17f5631.png)